react/week3/to-do-app/src/AddToDo.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function AddToDo({addNewTodo, todosLength}){ 
+    const [validDesc, setValidDesc] = useState(true);
+    const [validDate, setValidDate] = useState(true);
+    const [startDate, setStartDate] = useState(new Date());
+    const inFuture = (date) => {
+        const nowDate = new Date();
+        nowDate.setHours(0,0,0,0);
+        date.setHours(0,0,0,0);
+        return date.getTime()>=nowDate.getTime();
+      };
+      const onAddToDo = () => {
+        const newDescription=document.getElementById('newDescription').value;
```

Comment:
In general when using React it should not be necessary to use the "document.getElementById". In this case, it would be better to have a state variable that holds the text that is currently written in the input field. Then you could use that state instead of using "getElementById". Let me know if it makes sense, else I would be happy to show a more elaborate example.